addons/im_livechat/static/src/models/chatbot.js

```diff
@@ -2,16 +2,32 @@
 
 import { registerModel } from '@mail/model/model_core';
 import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'Chatbot',
     identifyingFields: ['livechatButtonViewOwner'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {integer}
```

Comment:
```suggestion
         * @returns {integer|FieldCommand}
```

Reply:
Let's see whether runbot's happy without guarding `this.data`. If not, then reintroduce the guard and make proper docstring.